recipes/hdf5-plugins/meta.yaml

```diff
@@ -0,0 +1,97 @@
+{% set name = "hdf5-plugins" %}
```

Comment:
This project seems to be called `hdf5-external-filter-plugins` ... kind of a mouthful, but it is probably important to keep the original name for discoverability

Reply:
@eugenwintersberger do you have an opinion on this?

Reply:
renamed